content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -1176,15 +1179,15 @@ the failure message.
 
     <!--pytest.mark.skip-->
 
-    ```sh
+    ```bash
     dotnet new console --name influxdb_csharp_client
     ```
 
 3.  Change into the generated `influxdb_csharp_client` directory.
 
     <!--pytest.mark.skip-->
 
-    ```sh
+    ```bash
```

Comment:
Should I start using `bash` instead of just `sh` (shell)?

Reply:
> Should I start using `bash` instead of just `sh` (shell)?

I'm leaning toward bash b/c it's arguably ranked as the most popular and it makes some tasks easier/prettier.